docs/configuring-the-deploy-cli.md

```diff
@@ -0,0 +1,135 @@
+
+### Environment variables
+
+By default, the Deploy CLI ingests environment variables, providing the ability to pass credentials and other configurations to the tool without needing to publish to the `config.json` file. Environment variables can either be used to augment the `config.json` file or replace it altogether depending on the project needs.
+
+Non-primitive configuration values like `AUTH0_KEYWORD_MAPPING` and `AUTH0_EXCLUDED` can also be passed in through environment variables so long as these values are properly serialized JSON.
+
+To **disable** the consumption of environment variables for either the `import` or `export` commands, pass the `--env=false` argument.
+
+#### Examples:
+
+```shell
+# Deploying configuration for YAML formats without a config.json file
+AUTH0_DOMAIN=<YOUR_AUTH0_DOMAIN> \
+AUTH0_CLIENT_ID=<YOUR_CLIENT_ID> \
+AUTH0_CLIENT_SECRET=<YOUR_CLIENT_SECRET> \
+a0deploy import --input_file=local/tenant.yaml
+
+# Disable environment variable ingestion
+a0deploy export -c=config.json --format=yaml --output_folder=local --env=false
+
+# Non-primitive configuration values
+AUTH0_EXCLUDED='["actions","organizations"]'\
+AUTH0_KEYWORD_REPLACE_MAPPINGS='{"ENVIRONMENT":"dev"}'\
+a0deploy export -c=config.json --format=yaml --output_folder=local
+```
+
+## Available Configuration Properties
+
+### `AUTH0_DOMAIN`
+
+String. The domain of the target Auth0 tenant.
+
+### `AUTH0_CLIENT_ID`
+
+String. The ID of the designated Auth0 application used to make API requests.
+
+### `AUTH0_CLIENT_SECRET`
+
+String. The secret of the designated Auth0 application used to make API requests.
+
+### `AUTH0_ACCESS_TOKEN`
+
+String. Short-lived access token for Management API from designated Auth0 application. Can be used in replacement to client ID and client secret combination.
+
+### `AUTH0_ALLOW_DELETE`
+
+Boolean. When enabled, will allow the tool to delete resources. Default: `false`.
+
+### `AUTH0_EXCLUDED`
+
+Array of strings. Excludes entire resource types from being managed, bi-directionally. See also: [excluding resources from management](#). Possible values: `actions`, `attackProtection`, `branding`, `clientGrants`, `clients`, `connections`, `customDomains`, `databases`, `emailProvider`, `emailTemplates`, `guardianFactorProviders`, `guardianFactorTemplates`, `guardianFactors`, `guardianPhoneFactorMessageTypes`, `guardianPhoneFactorSelectedProvider`, `guardianPolicies`, `hooks`, `logStreams`, `migrations`, `organizations`, `pages`, `prompts`, `resourceServers`, `roles`, `rules`, `rulesConfigs`, `tenant`, `triggers`
+
+#### Example
+
+```json
+{
+  "AUTH0_EXCLUDED": ["organizations", "connections", "hooks"]
+}
+```
+
+### `AUTH0_KEYWORD_REPLACE_MAPPINGS`
+
+Mapping of specific keywords to facilities dynamic replacement. See also: keyword replacement mapping.
```

Comment:
> See also: keyword replacement mapping.

This is missing the link

Reply:
This link is added in a downstream PR for the Keyword Replacement page. But was good to double check!